ui/assets/css/_req-filter-ui.scss

```diff
@@ -1,11 +1,15 @@
 @import '~react-select/scss/default';
 
 .req-filter-ui {
-  list-style: none;
 
-  li {
-    /* Handing indent */
-    padding-left: 15px;
-    text-indent: -15px;
+  .active-filter {
+    background-color: $color-gray-warm-light;
+    font-weight: bold;
+
+    .remove-filter-link {
```

Comment:
I've been warned pretty strongly against nesting selectors too deep. Can this be more general purpose?